lib/mixlib/install/version.rb

```diff
@@ -1,5 +1,5 @@
 module Mixlib
   class Install
-    VERSION = "0.8.0"
+    VERSION = "1.0.0"
```

Comment:
Actually, `0.8.0.pre`.  I'm accustomed to using tools like `gem-release` https://github.com/svenfuchs/gem-release where the version increments are automated.  Since we still at a 0 major version we should keep the increments small.  I think we should stay at 0.8 while we make these deeper cuts.